application/views/admin/global_settings/_email.php

```diff
@@ -6,91 +6,114 @@
-                "ssl"=>gT("SSL",'unescaped'),
-                "tls"=>gT("TLS",'unescaped')
-            )
-        ));?>
-    </div>
-</div>
-<div class="form-group">
-    <label class="  control-label"  for='emailsmtpdebug'><?php eT("SMTP debug mode:"); ?></label>
-    <div class="">
-        <?php $this->widget('yiiwheels.widgets.buttongroup.WhButtonGroup', array(
-            'name' => 'emailsmtpdebug',
-            'value'=> getGlobalSetting('emailsmtpdebug') ,
-            'selectOptions'=>array(
-                "0"=>gT("Off",'unescaped'),
-                "1"=>gT("On errors",'unescaped'),
-                "2"=>gT("Always",'unescaped')
-            )
-        ));?>
+        <div class="form-group">
+            <label class="  control-label"  for="emailsmtphost"><?php eT("SMTP host:"); ?></label>
+            <div class="">
+                <input class="form-control"  type='text' size='50' id='emailsmtphost' name='emailsmtphost' value="<?php echo htmlspecialchars(getGlobalSetting('emailsmtphost')); ?>" />
+                <p class="help-block"><?php printf(gT("Enter your hostname and port, e.g.: %s"),"smtp.example.org:25"); ?></p>
+            </div>
+        </div>
 
-        <br />&nbsp;
+        <div class="form-group">
+            <label class="  control-label"  for='emailsmtpuser'><?php eT("SMTP username:"); ?></label>
+            <div class="">
+                <input class="form-control"  type='text' size='50' id='emailsmtpuser' name='emailsmtpuser' value="<?php echo htmlspecialchars(getGlobalSetting('emailsmtpuser')); ?>" />
+            </div>
+        </div>
+
+        <div class="form-group">
+            <label class="  control-label"  for='emailsmtppassword'><?php eT("SMTP password:"); ?></label>
+            <div class="">
+                <input class="form-control"  type='password' autocomplete="off" size='50' id='emailsmtppassword' name='emailsmtppassword' value='somepassword' />
+            </div>
+        </div>
+
+        <div class="form-group">
+            <label class="  control-label"  for='maxemails'><?php eT("Email batch size:"); ?></label>
+            <div class="">
+                <input class="form-control"  type='text' size='5' id='maxemails' name='maxemails' value="<?php echo htmlspecialchars(getGlobalSetting('maxemails')); ?>" />
+            </div>
+        </div>
     </div>
-</div>
-<div class="form-group">
-    <label class="  control-label"  for='maxemails'><?php eT("Email batch size:"); ?></label>
-    <div class="">
-        <input class="form-control"  type='text' size='5' id='maxemails' name='maxemails' value="<?php echo htmlspecialchars(getGlobalSetting('maxemails')); ?>" />
+    
+    <div class="ls-flex-column ls-space padding left-5 right-5 col-md-5">
+
+        <!-- Test email -->
+        <div class="form-group">
+            <label class="text-left control-label" for='sendTestEmail'>
+            <?php eT("Send Test Email using saved options:"); ?>
```

Comment:
Should be 'Send test email using saved options:'

Reply:
Resolved in latest commit